src/modules/viewLikes.js

```diff
@@ -0,0 +1,20 @@
+import { Movies } from './resource.js';
+import likeCounter from './likeCounter.js';
+
+function displayLikes() {
+  const obj = new Movies();
+  obj.getLike().then((response) => {
+    if (response) {
+      response.forEach((element) => {
+        const like = document.getElementById(element.item_id);
+        const count = likeCounter(element.item_id, response);
+        if (like) {
+          const textNode = `<span>${count} likes </span>`;
+          like.innerHTML = textNode;
+        }
+      });
+    }
+  });
+}
```

Comment:
- Since you are using ES6, It would be better to use arrow functions instead of the word function.